sglang/code-walk-through/readme.md

```diff
@@ -101,7 +101,7 @@ The Scheduler continuously executes its event loop, alternating between `get_nex
 ### [run_batch](https://github.com/sgl-project/sglang/blob/f8b0326934bacb7a7d4eba68fb6eddebaa6ff751/python/sglang/srt/managers/scheduler.py#L956)
 
 1. For generation models, use TpModelWorker’s `forward_batch_generation` for token prediction or `forward_batch_idle` for idle tasks, returning results to `event_loop_normal`.
-2. For embedding or reward models, assert token extension, execute `forward_batch_embedding`, and return embeddings.
+2. For embedding or reward models, execute `forward_batch_embedding`, and return embeddings.
```

Comment:
assert token extension is so detailed, so we don't need to write this here - it's just a line of Python code.